CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "parisc.*")
 	set(HARDINFO_ARCH "parisc")
 elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "sparc{32,64}")
 	set(HARDINFO_ARCH "sparc")
+elseif(${CMAKE_HOST_SYSTEM_PROCESSOR} MATCHES "aarch64*")
+	set(HARDINFO_ARCH "aarch64")
```

Comment:
I've tested just using 'arm' on AArch64 (#72). It seems to work fine aside from what I think is an unrelated problem. I did notice that x86_64 uses symlinks to x86 instead, maybe it should be done the same way here.